LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthResult.kt

```diff
@@ -22,5 +22,5 @@ sealed class LocalPaymentAuthResult {
      * browser to cancel the payment flow, or returns to the app without completing the
      * authentication flow.
      */
-    object NoResult : LocalPaymentAuthResult()
+    data object NoResult : LocalPaymentAuthResult()
```

Comment:
Same change here let's remove `data` if we aren't using it across other result objects